t/plugin/basic-auth.t

```diff
@@ -398,7 +398,7 @@ GET /t
 
 
 
-=== TEST 15: enable basic auth plugin using admin api, set hide_credentials = true
+=== TEST 18: enable basic auth plugin using admin api, set hide_credentials = true
```

Comment:
Need to fetch the latest code for the master branch, which has been fixed.

Reply:
Done. Thanks